### DISCUSSION TITLE
How do I insert a blank line between two code blocks ?

### DISCUSSION THREAD

**Discussion by alkanna** [outline/outline#5966] [General] [answered]
Like here for instance :

![CleanShot 2023-10-09 at 00 40 17@2x](https://github.com/outline/outline/assets/30343764/357623c9-c08d-4c68-9a91-77b8d003df14)

Is there any way to insert a line between those two ?
I could not find a way to "break out" of a code block that would insert a new line that is not part of it. I also tried CTRL+SHIFT+\ but that just leads to the code block being converted to raw text, which is to be expected I assume.

**Comment by tommoor** [comment, marked as answer]
`Cmd`+`Enter` will exit the code block and create a paragraph, we should probably make it so that Enter on an empty line will exit too – the behavior between code and e.g blockquotes is a bit inconsistent right now

**Reply by alkanna** [reply]
Thanks !
I feel a bit stupid that I did not try that 😓

**Reply by alkanna** [reply]
@tommoor Created https://github.com/outline/outline/issues/5967 as enhancement issue about that.
